rdr_service/api/participant_summary_api.py

```diff
@@ -66,6 +70,33 @@ def _make_bundle(self, results, id_field, participant_id):
             return make_sync_results_for_request(self.dao, results)
         return super(ParticipantSummaryApi, self)._make_bundle(results, id_field, participant_id)
 
+    @staticmethod
+    def _check_constraints():
+        message = None
+        invalid = False
+
+        pairs = {
+            'lastName': {
+                'fields': ['lastName', 'dateOfBirth'],
+                'message': ''
```

Comment:
Does the message field here do anything?

Reply:
Good catch.